includes/class-wc-payments-tasks.php

```diff
@@ -30,8 +32,10 @@ public static function init() {
 	 * Adds a task to the WC 'Things to do next' task list the if disputes awaiting response.
 	 */
 	public static function add_task_disputes_need_response() {
+		include_once WCPAY_ABSPATH . 'includes/admin/tasks/class-wc-payments-task-disputes.php';
+
 		$account_service = WC_Payments::get_account_service();
-		if ( ! $account_service || ! $account_service->is_stripe_account_valid() ) {
+		if ( ! $account_service || ! $account_service->is_stripe_account_valid() || $account_service->is_account_under_review() || $account_service->is_account_rejected() ) {
```

Comment:
`is_account_under_review()` and `is_account_rejected` also check `is_stripe_connected`

Reply:
This could also be explained in a comment - i.e. define the preconditions in merchant/human terms :)